src/scenes/Event/WithdrawTo/index.js

```diff
@@ -1,92 +1,135 @@
 import React, { Fragment } from 'react';
 import { inject, observer } from 'mobx-react';
-import { FormattedMessage, injectIntl } from 'react-intl';
-import { TableBody, TableHead, TableRow, TableCell, Button } from '@material-ui/core';
+import { FormattedMessage, injectIntl, defineMessages } from 'react-intl';
+import { TableBody, TableRow, TableCell, Button, withStyles } from '@material-ui/core';
+import { makeStyles } from '@material-ui/core/styles';
 import { Warning, ResponsiveTable } from 'components';
 import Icon from '../Icon';
 import Container from '../Container';
 import Label from '../Label';
 import { i18nToUpperCase } from '../../../helpers/i18nUtil';
 import { toFixed } from '../../../helpers/utility';
+import styles from './styles';
+
+
+const useTableRowStyles = makeStyles(() => ({
+  root: {
+    border: '1px solid rgba(151, 151, 151, 0.1)',
+  },
+}));
+
+const messages = defineMessages({
+  escrow: {
+    id: 'str.escrow',
+    defaultMessage: 'Escrow',
+  },
+  totalBet: {
+    id: 'str.totalBet',
+    defaultMessage: 'Your Total Bet',
+  },
+  totalBetReturn: {
+    id: 'str.totalBetReturn',
+    defaultMessage: 'Your Total Bet Return',
+  },
+  totalVote: {
+    id: 'str.totalVote',
+    defaultMessage: 'Your Total Vote',
+  },
+  totalVoteReturn: {
+    id: 'str.totalVoteReturn',
+    defaultMessage: 'Your Total Vote Return',
+  },
+  totalReturn: {
+    id: 'str.totalReturn',
+    defaultMessage: 'Your Total Return',
+  },
+});
 
 const WithdrawTo = observer(({ store: { eventPage: { withdrawableAddress }, wallet: { currentAddress } } }) => {
   if (currentAddress !== '') {
     return (<Container>
       <WalletIcon />
       <WithdrawToLabel />
       <WithdrawList withdrawableAddress={withdrawableAddress} />
+      <WithdrawButton />
     </Container>);
   }
   return <Fragment />;
 });
 
 const WalletIcon = () => <Icon type='wallet' />;
 
+const FormattedMessageFixed = (props) => <FormattedMessage {...props} />;
+
 const WithdrawToLabel = () => (
   <Label>
-    <FormattedMessage id="withdrawDetail.withdrawTo" defaultMessage="WITHDRAW TO">
+    <FormattedMessage id="withdrawDetail.detail" defaultMessage="Detail">
       {(txt) => i18nToUpperCase(txt)}
     </FormattedMessage>
   </Label>
 );
 
-const WithdrawList = ({ withdrawableAddress }) => (
+const WithdrawButton = withStyles(styles)(inject('store')(observer(({ store, classes }) => {
+  const { eventPage: { didWithdraw, pendingWithdraw, withdraw } } = store;
+  const { id, message, warningType, disabled } = getActionButtonConfig(
+    pendingWithdraw,
+    didWithdraw,
+  );
+
+  return (
+    <Fragment>
+      <Button
+        size="small"
+        variant="contained"
+        color="primary"
+        disabled={disabled}
+        className={classes.button}
+        onClick={() => withdraw()}
```

Comment:
nit: if `withdraw()` is using fat-arrow syntax you can just pass in `{withdraw}`